src/replicache.ts

```diff
@@ -78,31 +75,56 @@ export default class Replicache {
     await p;
   }
 
-  private _invoke<Rpc extends keyof InvokeMapNoArgs>(
-    rpc: Rpc,
-  ): Promise<InvokeMapNoArgs[Rpc]>;
-  private _invoke<Rpc extends keyof InvokeMap>(
-    rpc: Rpc,
-    args: InvokeMap[Rpc][0],
-  ): Promise<InvokeMap[Rpc][1]>;
-  private async _invoke(rpc: string, args?: JsonType): Promise<JsonType> {
+  private _invoke: Invoke = async (
+    rpc: string,
+    args?: JsonType,
+  ): Promise<JsonType> => {
     await this._opened;
     return await this._repmInvoke(this._name, rpc, args);
+  };
+
+  /** Get a single value from the database. */
+  get(key: string): Promise<JsonType> {
+    return this.query(tx => tx.get(key));
+  }
+
+  /** Determines if a single key is present in the database. */
+  has(key: string): Promise<boolean> {
+    return this.query(tx => tx.has(key));
+  }
+
+  /** Gets many values from the database. */
+  scan({prefix = '', start, limit = 50}: ScanOptions = {}): Promise<
```

Comment:
Totally unrelated, but someday it would be nice to do a proper scan / iterator API.